### ISSUE TITLE
piper tts pauses after each line when using NVDA addon

### ISSUE THREAD

**Issue by joshknnd1982** [rhasspy/piper#94]
I noticed the latest piper tts NVDA beta addon still pauses at the end of each line. It does not read complete sentences, can this be fixed? thanks.
